tests/integration/test_integration_vespa_cloud_token.py

```diff
@@ -223,13 +228,17 @@ def setUp(self) -> None:
             build_status = self.vespa_cloud.check_production_build_status(
                 build_no=self.build_no
             )
-            if build_status["status"] == "done":  # TODO:  add build_status["deployed"]:
-                success = True
-                break
+            if build_status["status"] == "done":
+                if build_status["deployed"]:
+                    success = True
+                    break
```

Comment:
If `status` is `done`, then I guess there's not point in polling again, regardless of whether it's a success. So perhaps provide a dedicated message explaining that it _never_ will succeed, as opposed to the wait timing out. In both cases, a link to the deployment overview in the UI would be great, if that's readily available from `pyvespa`. (It should be, if it isn't 😝)

Reply:
Oh this was still in the test class 😆 Well, the point about not polling further still stands, although I guess this situation is unlikely to occur in the test scenario.

Reply:
Good point! 
The link to the deployment in console is shown in `deploy_to_prod()`. 
Could probably add separate function - `wait_for_production_deployment(max_wait: int)`. As this is probably something the user will want to do anyway (and we in our tests).

Reply:
Sounds like a good idea!